src/client/MainPage.react.js

```diff
@@ -11,17 +11,25 @@ const styles = require('./styles.css');
 type Props = {};
 
 function initConnection(userId: string) {
+  // TODO: proper onClick input validation
+  if (userId == '') {
+    alert("userID cannot be blank");
+    return false;
+  }
```

Comment:
in the future this can go before line 42, so it's separate from the `setConn` function.